binders.tex

```diff
@@ -1,12 +1,14 @@
 \begin{syntax}
   binders ::= binder (, binder)* ;
   \
-  binder ::= type-expr variable-ident;
+  binder ::= type-name variable-ident;
```

Comment:
I don't understand this change. As far as I can tell, this would prevent writing something like
`assert list_positive: \forall list* p; reachable(root, p) ==> p->content > 0;` unless we have an explicit `typedef` for `list *`.

Reply:
OK, back on the topic after a too long hiatus. You were [right](https://github.com/acsl-language/acsl/pull/62#issuecomment-515016081), `variable-indent` takes care of qualifiers as in C. I don't know why I thought otherwise.